api/src/main/java/io/opentelemetry/trace/DefaultSpan.java

```diff
@@ -31,19 +32,21 @@
 @Immutable
 public final class DefaultSpan implements Span {
 
+  private static final Random random = new Random();
+
   /**
    * Creates an instance of this class.
    *
-   * <p>Each instance will have a unique and valid {@link SpanId}.
+   * <p>Each instance will have unique and valid {@link TraceId} and {@link SpanId}.
    *
    * @return a {@link DefaultSpan}.
    * @since 0.1.0
    */
   public static DefaultSpan create() {
```

Comment:
why is this public?

Reply:
Previously `BlankSpan.INSTANCE` is public, and it got converted to `DefaultSpan.create()` per https://github.com/open-telemetry/opentelemetry-java/issues/309 and https://github.com/open-telemetry/opentelemetry-java/issues/310#issuecomment-498797101. I don't think these issues mentioned this should be removed from API - or should it?

Reply:
BlankSpan.INSTANCE is different than DefaultSpan.create(). We may have a getInvalid or something like that on the DefaultSpan to replace that functionality.

Reply:
It's also used in a different package `trace.unsafe`: https://github.com/open-telemetry/opentelemetry-java/blob/f62dbff017c7caea566f3fbb0959343b2b5265f8/api/src/main/java/io/opentelemetry/trace/unsafe/ContextUtils.java#L35. I think before we have the `getInvalid` API, `create()` needs to be public. Otherwise we need to set the default value of `CONTEXT_SPAN_KEY` to null.

Reply:
The default result if no Span associated with the context definitely should not be a Random trace/span id. Should be one with Invalid SpanContext.

Reply:
Make sense, in this case I think we need the `getInvalid` API. Updated in https://github.com/open-telemetry/opentelemetry-java/pull/382/commits/19f15476c932b4f8cdda13664d208d5bd42d1b18.